.eslintrc.json

```diff
@@ -23,56 +22,79 @@
   "extends": [
     "eslint:recommended",
     "plugin:import/recommended",
-    "plugin:import/typescript",
     "prettier",
     "plugin:prettier/recommended",
-    "plugin:@typescript-eslint/recommended",
-    "plugin:@typescript-eslint/recommended-requiring-type-checking",
     "plugin:json/recommended"
   ],
   "rules": {
-    "no-restricted-syntax": [
-      "error",
-      {
-        "selector": "TSEnumDeclaration[const=true]",
-        "message": "const enums are not supported in isolated module mode"
-      }
-    ],
-
     "import/default": ["warn"],
     "import/no-cycle": ["warn"],
     "import/no-duplicates": "off",
     "simple-import-sort/imports": "error",
     "simple-import-sort/exports": "error",
-    "@typescript-eslint/consistent-type-imports": [
-      "error",
-      {
-        "prefer": "type-imports"
-      }
-    ],
-    "@typescript-eslint/consistent-type-exports": [
-      "error",
-      {
-        "fixMixedExportsWithInlineTypeSpecifier": true
-      }
-    ],
-    "@typescript-eslint/no-unused-vars": [
-      "warn",
-      {
-        "vars": "all",
-        "args": "after-used",
-        "ignoreRestSiblings": false,
-        "caughtErrors": "none",
-        "argsIgnorePattern": "^_"
-      }
-    ],
     "unused-imports/no-unused-imports": "error",
     "no-empty": "warn",
     "no-fallthrough": "off",
     "prefer-const": "off",
     "prefer-arrow-callback": "warn"
   },
   "overrides": [
+    {
+      "files": ["**/*.{ts,tsx}"],
```

Comment:
it turned out a good chunk of the errors in the lint command about tsconfig.json / project were due to running these lints on file extensions that were not ts or tsx as those other files are excluded from the tsconfig.json (js, d.ts, etc)

Reply:
Great catch!